### PR TITLE
Update README.md: To include OSX instructions to add `source dnvm.sh` to bashrc - to use in bash

### PR DESCRIPTION
**Pull request by MRdNk**
To show bash profile step for osx #499

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@MRdNk__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, MSOTBOT;

**Comment by Eilon** [comment]
@glennc can you review?

File: README.md [hunk 1]
```diff
@@ -47,6 +47,13 @@ brew tap aspnet/dnx
 brew update
 brew install dnvm
 ```
+
+Add dnvm to your bash profile (./bash_profile)
+```bash
+source dnvm
```
Comment (glennc) [inline README.md]: `source dnvm.sh`? or does that work without the extension?

Comment (henkmollema) [inline README.md]: Does needs the extension: `-bash: dnvm: No such file or directory`.

**Comment by jasonwilliams200OK** [comment]
@MRdNk, would be nice if the title of this PR and the other one is updated to something more specific rather than "Update README.md". It doesn't tell others much what has been updated. BTW, the title is editable, try it on :-}~

**Comment by MRdNk** [comment]
@jasonwilliams200OK yeah sorry, my bad! I'd should follow the rules I set for my devs.

**Comment by glennc** [comment]
Sorry this took a while for me to get to. Thanks for the PR :)

**Pull request merged**